shared/teams/team/new-header.tsx

```diff
@@ -47,6 +47,7 @@ const FeatureTeamCard = ({teamID}: FeatureTeamCardProps) => {
       className="addInviteAndLinkBox"
       alignItems="flex-start"
       alignSelf="flex-end"
+      fullWidth={true}
```

Comment:
Did you test this on desktop?

Reply:
Yeah looks good on desktop with it as far as i can tell!